tests/ui/structs/default-field-values/non-exhaustive-ctor.rs

```diff
@@ -0,0 +1,26 @@
+#![feature(default_field_values)]
```

Comment:
Discussion: also note that this test does not snapshot the other suggestion where we are nightly-only but do not have the feature gate. Maybe revision this test, but I don't think it matters that much.

Reply:
I can add that.